### ISSUE TITLE
Issues creating I2A Classrooms

### ISSUE THREAD

**Issue by shaunanoordin** [zooniverse/edu-api-front-end#72]
I just caught this while I was checking out PR #69 (tee hee) but I can't be sure: when this issue was introduced, whether it's a change on the app code or the Edu API server, or whether it's a temporary (service) issue on the staging server.

## Issue
When creating an I2A Classroom (on localhost+OSX+Chrome as `zootester1`), two unexpected things happen:
- The Create Classroom modal doesn't close
- The three I2A Assignments aren't auto-created

![screen shot 2017-10-16 at 14 32 16](https://user-images.githubusercontent.com/13952701/31614947-e43fcd34-b27f-11e7-93ad-03f476f585ae.png)

### Observations
- Process is as follows:
  1. sign in as `zootester1` on localhost, then go to I2A
  2. Click "Create" (Classroom), then fill in a name for the classroom (`zooterter1 test - 16 Oct 2017`), the click OK ("Create").
  3a. Expected: modal should close. Classroom Manager should show newly created classroom with 3 I2A Assignments.
  3b. Actual: modal doesn't close. Error message appears: "Something went wrong". **Classroom is created,** but it has no pre-generated I2A Assignments.
- In EduAPI, `/assignments?classroom_id={RECENTLY_CREATED_CLASSROOM}` seems to issue a 500. `/classrooms` and `/assignments?classroom_id={OTHER_CLASSROOM}` seem to be OK though.
- The modal not closing may be _unrelated_ to the issue with the `/assignment` endpoint.

### Status
Investigating

**Comment by srallen** [comment]
The payload sent with the assignment creation needs updating since the PR for programs got merged. It no longer requires the program id, since the classroom should be associated with the program. I started to work on updating this, but uncovered a minor bug with the API that should be fixed in the incoming zooniverse/education-api#70.

Thanks for documenting this as it is a good reminder that this is still needing fixing.
